.circleci/config.yml

```diff
@@ -1188,7 +1188,7 @@ jobs:
           command: |
             echo "Checking all contracts under ./artifacts"
             docker run --volumes-from with_code rust:1.59.0 \
-              /bin/bash -e -c 'export GLOBIGNORE="../../artifacts/floaty.wasm"; cd ./code/packages/vm; ./examples/check_contract.sh ../../artifacts/*.wasm'
+              /bin/bash -e -c 'export GLOBIGNORE="../../artifacts/floaty.wasm"; cd ./code/packages/vm; cargo install cw-check-contract; cw-check_contract ../../artifacts/*.wasm'
```

Comment:
This installs an external dependency. We should use the tool from the local repository.

Reply:
Ah, yeah. I think we still need to move `cw-check-contract` from [`cw-tools`](https://github.com/CosmWasm/cw-tools) to this repo and then we can use that.

Reply:
@jawoznia Are you okay doing that? As per https://github.com/CosmWasm/cosmwasm/issues/1371#issuecomment-1216562000